test/benchmarks/ramanujan_pi.c

```diff
@@ -0,0 +1,34 @@
+#include <math.h>
+
+#define WASM_EXPORT __attribute__((visibility("default")))
+
+static unsigned factorial(unsigned n)
```

Comment:
Can be also implemented with `double n`.

Reply:
Yeah thought about those, but kind of resorted that perhaps for forcefully having "more floating point" perhaps a hand crafted synthetic benchmark using all ops is better?

Reply:
So what's the point of this benchmark if only 10-20% of instructions is floating-point?

Reply:
Absolute number of instructions or instructions weighted by the number of times they are executed?

If you profile it it should be clear the heavy part is the floating point calculation (pow, etc.). The factorial part only works with small number (<30).

Reply:
Ok then.